config/crd/bases/anywhere.eks.amazonaws.com_bundles.yaml

```diff
@@ -1659,8 +1659,6 @@ spec:
                               description: The image repository, name, and tag
                               type: string
                           type: object
-                        version:
```

Comment:
I didn't change this, I assume this is because we never regenerated the manifests after removing that field
@abhay-krishna